src/region/RegionAnnotations.tsx

```diff
@@ -63,7 +63,7 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
 
     handleStop = (shape: Rect): void => {
         const { location, setStaged, setStatus } = this.props;
-        setStaged({ location, shape });
+        setStaged({ target: { location: { value: location, type: 'page' }, shape, type: 'region' } });
```

Comment:
This was intentionally left to the action to prevent the component from needing to be aware of it. Let's revert this change.